community/maven/APKBUILD

```diff
@@ -2,13 +2,14 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=maven
-pkgver=3.6.0
+pkgver=3.6.1
 _pkgname="$pkgname-${pkgver%%.*}"
 pkgrel=0
 pkgdesc="A Java project management and project comprehension tool."
 url="http://maven.apache.org"
 arch="noarch"
 license="Apache-2.0"
+options="!check"
```

Comment:
it needs to add comment why tests are disabled

Reply:
note added